web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.html

```diff
@@ -118,11 +118,13 @@
 </div>
 
 <convoy-card>
-	<div class="flex">
+	<convoy-empty-state type="table" imgSrc="/assets/img/events-log-empty-state.png" className="min-h-[70vh]" heading="You currently do not have any event logs" *ngIf="!isloadingEvents && (!displayedEvents || displayedEvents?.length === 0)"></convoy-empty-state>
```

Comment:
Empty state components, please lets rewrite them into a directive instead. Lets take out the use of `className` as much as possible